src/translate.py

```diff
@@ -4372,6 +4373,16 @@ def address_of_gsym(self, sym_name: str) -> AddressOf:
 
         return AddressOf(sym, type=sym.type.reference())
 
+    def flow_graph(self, function: Function) -> FlowGraph:
```

Comment:
might be cleaner to pass this as an argument to translate_to_ast rather than memoize it